content/aips/AAVE_ETHEREUM_V3.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Aave Ethereum V3
+author: BGD Labs (@bgdlabs)
+shortDescription: Activation of Aave v3 Ethereum
+discussions: https://governance.aave.com/t/bgd-aave-v3-ethereum-new-deployment-vs-aave-v2-upgrade/9990/16
+created: 2023-01-20
+---
+
+# Simple Summary
+
+This governance proposal activates the Aave V3 Ethereum pool (3.0.1), by completing all the initial setup and listing WBTC, WETH, wstETH, USDC, DAI, LINK, and AAVE, already pre-approved by the community.
+All the Aave Ethereum V3 addresses can be found in the [aave-address-book](https://github.com/bgd-labs/aave-address-book/blob/main/src/AaveV3Ethereum.sol).
+
+# Motivation
+
+While AAVE V3 was deployed to various networks right after the release, the Ethereum pool was still running V2.
+After careful consideration, the community [decided to deploy a fresh V3](https://snapshot.org/#/aave.eth/proposal/0x584eb4e0f79e1d9dcdd99b3a0c831bfc3c654af3f8f619d5f68eae23cd9cb149) instead of upgrading the V2 pool, for increased compatibility between the V3 pools and less general complexity.
+The assets to be listed & respective configurations have been discussed in [the governance forum](https://governance.aave.com/t/arc-aave-v3-ethereum-deployment-assets-and-configurations/10238) and been decided in the following [snapshot](https://snapshot.org/#/aave.eth/proposal/0xc31254fac1369090cea7c0105cbc6381b72189c038391996f855708ff2e0c02e).
+
+# Specification
+
+This proposal's payload does the following:
+
+1. Create the eMode category for eth correlated assets via `POOL_CONFIGURATOR.setEModeCategory(1, 90_00, 93_00, 10_100, address(0), 'ETH correlated')`
+2. Swap of emergency admin to Aave Guardian via `ACL_MANAGER.removeEmergencyAdmin(AaveGovernanceV2.SHORT_EXECUTOR)` followed by `ACL_MANAGER.addEmergencyAdmin(GUARDIAN_ETHEREUM)`
+3. grant `POOL_ADMIN` permissions to all the asset listing stewards created inside the `constructor()` via `ACL_MANAGER.addPoolAdmin(stewards[i])`
+
+The Steward pattern used in [V3-listing stewards](https://github.com/bgd-labs/aave-v3-listing-stewards/blob/feat/v3-ethereum-tests/src/contracts/common/StewardBase.sol#L8). Each Steward will handle the listing of one specific asset and then renounce to the granted POOL_ADMIN role. This process is needed as otherwise, the proposal's gas limit would be really gas-expensive.
```

Comment:
```suggestion
The Steward pattern is taken from [V3-listing stewards](https://github.com/bgd-labs/aave-v3-listing-stewards/blob/feat/v3-ethereum-tests/src/contracts/common/StewardBase.sol#L8). Each Steward will handle the listing of one specific asset and then renounce to the granted POOL_ADMIN role. This process is needed as otherwise, the proposal's gas limit would be really gas-expensive.
```